packages/core/src/App/app.jsx

```diff
@@ -49,11 +49,15 @@ const AppWithoutTranslation = ({ root_store }) => {
 
         initializeTranslations();
         if (process.env.RUDDERSTACK_KEY) {
-            Analytics.initialise({
-                growthbookKey: process.env.GROWTHBOOK_CLIENT_KEY,
-                growthbookDecryptionKey: process.env.GROWTHBOOK_DECRYPTION_KEY,
+            const config = {
+                growthbookKey:
+                    process.env.IS_GROWTHBOOK_ENABLED === 'true' ? process.env.GROWTHBOOK_CLIENT_KEY : undefined,
+                growthbookDecryptionKey:
+                    process.env.IS_GROWTHBOOK_ENABLED === 'true' ? process.env.GROWTHBOOK_DECRYPTION_KEY : undefined,
```

Comment:
growthbook is optional in current package implementation. why do we need a separate variable?

Reply:
this is to disable the growthbook entirely from a feature flag registry. 
so we dont need to use any release to disable a thirdparty service. 
moving forward we have to do this for all of the thirdparty integrations.

Reply:
thanks @ali-hosseini-deriv